docs/how-to/create-a-custom-package.md

```diff
@@ -0,0 +1,115 @@
+# How to Create a Custom Package
+
+## Context
+
+The `/expo` and `/next` apps use the custom packages inside `/packages` to import shareable code between the Mobile and Web applications such as `packages/frontend/ui`. This package allows developers to create a single component that can be shared between platforms
+
+## Guide
+
+### 1. Create your custom package
+
+First create a directory for your new package:
+
+```
+$ mkdir example
+```
+
+Then initialise your new package:
+
+```
+$ yarn init
+```
+
+Then make sure to name your package appriopriately. Ensure you are extending off the existing Eslint config (and Babel if you need to compile JS in your package).
+
+Use the `package.json` template below as a starting point:
+
+```json
+// package.json
+
+{
+  "version": "0.0.0",
+  "name": "@frontend/example",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/types/index.d.ts",
+  "files": ["dist"],
+  "scripts": {
+    // ...
+  },
+  "dependencies": {
+    // ...
+  },
+  "peerDependencies": {
+    // ...
+  },
+  "devDependencies": {
+    // ...
+    "@babel/cli": "x.x.x",
+    "@shared/babel-config": "*",
+    "@shared/eslint-config": "*",
+    "@shared/tsconfig": "*",
+    "rimraf": "x.x.x",
```

Comment:
Is this required? Would keep this trimmed to everything you need

Reply:
For packages that need compiled JS I do think so, that's why I added them in. I can make it clear that this is for those specific type of packages